backend/core/serverless.ts

```diff
@@ -4,8 +4,8 @@ import { AWS } from '@serverless/typescript';
 import { httpApiResourceContract } from '@sls-monorepo/core-contracts';
```

Comment:
why is `yarn.lock` modified in this commit ?

Reply:
Hmm I guess it must be a rebase issue